templates/static/accessibility.html

```diff
@@ -4,7 +4,21 @@
 {% block main %}
   {% do page | setAttribute("feedback", "") %}
 
-  <h1>{{ _("Census questionnaire accessibility statement") }}</h1>
+  {% if schema_theme == "census-nisra" %}
+
+  {% set contact_us_url = "https://census.gov.uk/ni/contact-us/" %}
+
+{% elif language_code == "cy" %}
+
+  {% set contact_us_url = "https://cyfrifiad.gov.uk/cysylltu-a-ni/" %}
+
+{% else %}
```

Comment:
The indentation on these if statements is incorrect. Can't we set the url outside the if to the default and drop one of the switches?

Wouldn't it make more sense to set this url in the backend?